src/core/or/or.h

```diff
@@ -834,6 +837,10 @@ typedef struct protover_summary_flags_t {
    * service rendezvous point supporting version 3 as seen in proposal 224.
    * This requires HSRend=2. */
   unsigned int supports_v3_rendezvous_point: 1;
+
+  /** True iff this router has a protocol list that allows clients to
+   * negotiate link-level padding. Requires Padding=1. */
```

Comment:
Shouldn't this be "Padding >= 1"?

Reply:
e7be646f54cbc603ee2a74fe09277cbf8c0bb11e. Tho technically the code as-is checks to see if Padding=1 is present in the protover.. But I think that's right. If we bump padding in a way that is backwards-compatible, we can do Padding=1-2. Otherwise we'll do Padding=2.

(Sadly, if we don't get consensus serialization into 0.3.6, we'll have to do Padding=2 for that, not Padding=1-2, since Padding=1 relays won't read the consensus).